src/components/BlogPostCard/index.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import "./style.scss";
+
+import { LoveIcon } from "components/Icons/LoveIcon";
+import { ChatIcon } from "components/Icons/ChatIcon";
+import { BookMarkIcon } from "components/Icons/BookMarkIcon";
+import { Avatar } from "../Avatar";
+const BlogPostCard = ({
+  heading,
+  content,
+  image,
+  imageTitle,
+  authorImg,
+  date = "May 17",
+  authorName = "XettriAl.",
```

Comment:
Does not make much sense to have default values for props `date` and `authorName` as "May 17" and "XettriAl.".